src/main.ts

```diff
@@ -37,10 +43,10 @@ export function main(command?: string) {
       : () => myAdventures() === -args.turns;
 
   let digitizes = -1;
-  const globeTheater = $location`Globe Theatre Main Stage`;
   const yrTarget = $location`The Cave Before Time`;
 
-  const quest: ChronerQuest = { ...rose, completed };
+  const quest: ChronerQuest =
+    args.mode === "capsule" ? { ...capsule, completed } : { ...rose, completed };
```

Comment:
Do we plan on adding more modes in the future? if so, we'll need another structure than iterated ternary. I have a vision in my head for it when it comes to that, though.